src/app/rosetta/lib/block.ml

```diff
@@ -416,16 +441,23 @@ WITH RECURSIVE chain AS (
 
     let query =
       Caqti_request.collect Caqti_type.int typ
-        {| SELECT u.id, u.comamnd_type, u.fee_payer_id, u.source_id, u.receiver_id, u.nonce, u.amount, u.fee,
+        {| SELECT u.id, u.command_type, u.fee_payer_id, u.source_id, u.receiver_id, u.nonce, u.amount, u.fee,
         u.valid_until, u.memo, u.hash,
         pk1.value as fee_payer, pk2.value as source, pk3.value as receiver,
         blocks_user_commands.status,
-        blocks_user_commands.failure_reason
+        blocks_user_commands.failure_reason,
+        ac_payer.creation_fee,
+        ac_receiver.creation_fee
         FROM user_commands u
         INNER JOIN blocks_user_commands ON blocks_user_commands.user_command_id = u.id
-        INNER JOIN public_keys pk1 ON pk1.id = u.fee_payer_id
-        INNER JOIN public_keys pk2 ON pk2.id = u.source_id
-        INNER JOIN public_keys pk3 ON pk3.id = u.receiver_id
```

Comment:
Since aliases for these joined tables changed, they must also be updated above in line `446`.

Reply:
You're right. I was working on the queries outside the OCaml code, so I missed pulling those changes back here.